documentation/shared/attributes.adoc

```diff
@@ -43,13 +43,13 @@
 
 //OAuth attributes and links
 :oauth2-site: link:https://oauth.net/2/[OAuth 2.0 site^]
-:oauth-version: 0.7.1
```

Comment:
Why do you remove this? Just use this instead of the `OAuthTag` or rename it to `OAuthVersion`. It also needs to stay set to the version used by Strimzi. And not to `main`.

Reply:
It's better to not have it hardcoded but set from outside. There is `OAuthVersion` now, that replaces `OAuthTag`.